packages/expo-updates/CHANGELOG.md

```diff
@@ -32,6 +32,7 @@
 - [Android] Fixed the `node` execution on Windows. ([#23983](https://github.com/expo/expo/pull/23983) by [@kudo](https://github.com/kudo))
 - Bare update manifest non-nullability parity. ([#23166](https://github.com/expo/expo/pull/23166) by [@wschurman](https://github.com/wschurman))
 - Support importing assets from out of the project root when working in monorepos. ([#24090](https://github.com/expo/expo/pull/24090) by [@EvanBacon](https://github.com/EvanBacon))
+- Prevent failed updates from passing checkForUpdateAsync(). ([#24112](https://github.com/expo/expo/pull/24112) by [@douglowder](https://github.com/douglowder))
```

Comment:
Depending on our definition of breaking changes, because this changes the type names it's technically a breaking change. Fine leaving it here in bug fixes as well though. Either way, I'm in favor of the new names.

Reply:
Hopefully not breaking, I left the old names in and marked them deprecated.